app/views/sessions/new.html.erb

```diff
@@ -1,3 +1,5 @@
+<div class="header-buffer"></div>
```

Comment:
Imagine nagging every dev to add this to their views when they forget! Nightmare! Can it go somewhere in the layout? Near the nav?